includes/REST_API/Products_Controller.php

```diff
@@ -155,7 +155,14 @@ public function get_items( $request ) {
 		$query             = $this->shopping_vendors->get_vendor_class( $shopping_provider );
 
 		if ( ! $query ) {
-			return new WP_Error( 'unable_to_find_class', __( 'Unable to find class', 'web-stories' ), [ 'status' => 400 ] );
+			$shopping_provider = \strval( $this->settings->get_setting( Settings::SETTING_NAME_SHOPPING_PROVIDER ) );
```

Comment:
Here's the lint error

<img width="773" alt="Screen Shot 2022-06-01 at 5 52 52 AM" src="https://user-images.githubusercontent.com/62242/171378037-2c87abe7-cfee-4bd7-803d-9134b7053209.png">

Reply:
Added an ignore --- for that line

Reply:
@timarney Instead of adding an ignore, what we usually do elsewhere is to add a docblock comment to tell PHPStan about the expected type

```php
/**
 * Shopping provider.
 *
 * @var string $shopping_provider
*/
$shopping_provider = $this->settings->get_setting( Settings::SETTING_NAME_SHOPPING_PROVIDER );
```

What I do not understand about this code is why you are getting this setting _again_ when this is already done 5 lines above?

Reply:
hmmm --- yeah I just removed the line .... maybe got in there from a merge commit or something.  For sure not needed.